app/assets/javascripts/tylium/pages/activities.js

```diff
@@ -0,0 +1,62 @@
+document.addEventListener('turbolinks:load', function() {
+  var $activitiesGroupsContainer = $('[data-behavior="activities-groups-container"]');
+  var loading = false;
+  var canLoadMore = true;
+
+  if ($activitiesGroupsContainer.length) {
+    var $viewContent = $('[data-behavior="view-content"]');
+
+    $viewContent.on('scroll', function() {
+      var viewHeight = $viewContent.height();
+      var scrollTop = $viewContent.scrollTop();
+      var scrollHeight = $viewContent[0].scrollHeight;
+
+      // 64px is the sum of margin bottom and padding bottom of the content-container class
+      if (canLoadMore && !loading && viewHeight + scrollTop >= (scrollHeight - 64)) {
+        var page = $activitiesGroupsContainer.data('page') + 1;
+        var url = $activitiesGroupsContainer.data('url');
+        $('[data-behavior="activities-spinner"]').show();
+        loading = true;
+
+        $.ajax({
+          url: url,
+          data: { page: page },
+          success: function(data, status, XHR) {
```

Comment:
```suggestion
          success: function(data) {
```

Or are we using the other 2 arguments?

Reply:
No we're not, i'll remove them